voucher/src/main/java/com/adyen/checkout/voucher/VoucherView.kt

```diff
@@ -90,6 +92,12 @@ internal class VoucherView @JvmOverloads constructor(
 
     override fun getView(): View = this
 
+    private fun loadLogo(paymentMethodType: String?) {
```

Comment:
Double checking now, I think I removed these 2 methods by mistake while refactoring:
```kotlin


    private fun launchDownloadIntent() {
        val url = component.outputData?.downloadUrl ?: return
        val intent = CustomTabsIntent.Builder()
            .setShowTitle(true)
            .setToolbarColor(ThemeUtil.getPrimaryThemeColor(context))
            .build()
        intent.launchUrl(context, Uri.parse(url))
    }

    private fun loadLogo(paymentMethodType: String?) {
        if (!paymentMethodType.isNullOrEmpty()) {
            imageLoader.load(paymentMethodType, binding.imageViewLogo, LogoApi.Size.MEDIUM)
        }
    }
```

Should we re-add the other one as well?

Reply:
Yes, because now there is no way to download the pdf anymore 😆 I will add it